zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -216,23 +223,21 @@ where
                     let new_addrs = rsp_addrs
                         .iter()
                         .filter(|meta| !address_book.lock().unwrap().contains_addr(&meta.addr))
```

Comment:
We can delete this filter and close #1976, it's all implemented inside the address book now